### PR TITLE
Some tests fixture

### LINKED ISSUE #39: Fix lint errors, add lint check to test pipeline
After build will be green again https://github.com/Lingvokot/lingvokot.github.io/pull/27

### LINKED ISSUE #50: Fix tests errors and warnings
1. Fix 404 warnings
2. Fix timeout exceeding

### PR DESCRIPTION
**Pull request by keshan3262**
Close #50 
Close #39

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,13 +5,14 @@
   "description": "Static website hosten on GH Pages build with modern FrontEnd stack",
   "main": "index.js",
   "scripts": {
+    "test": "npm run test:karma:sauce & npm run lint",
```
Comment (ColCh) [inline package.json]: not `&` but `&&` should be there

File: .eslintcache [hunk 2]
```diff
@@ -0,0 +1 @@
+{"/home/keshan/lingvokot/lingvokot.github.io/src/components/Footer.js":{"size":1757,"mtime":1492705229254,"hashOfConfig":"3965ce17149c3ed309685dee62a22647","results":{"filePath":"/home/keshan/lingvokot/lingvokot.github.io/src/components/Footer.js","messages":[],"errorCount":0,"warningCount":0}},"/home/keshan/lingvokot/lingvokot.github.io/src/components/Screens/ForInvestors.js":{"size":1247,"mtime":1492702390783,"hashOfConfig":"3965ce17149c3ed309685dee62a22647","results":{"filePath":"/home/keshan/lingvokot/lingvokot.github.io/src/components/Screens/ForInvestors.js","messages":[],"errorCount":0,"warningCount":0}},"/home/keshan/lingvokot/lingvokot.github.io/src/index.js":{"size":730,"mtime":1492701474165,"hashOfConfig":"3965ce17149c3ed309685dee62a22647","results":{"filePath":"/home/keshan/lingvokot/lingvokot.github.io/src/index.js","messages":[],"errorCount":0,"warningCount":0}}}
```
Comment (ColCh) [inline .eslintcache]: Oh, sorry, forgot to mention, that this file should be ignored by git (add it to `.gitignore`)

File: base [hunk 3]
```diff
@@ -0,0 +1 @@
+.
```
Comment (ColCh) [inline base]: What is this file? Check it. I suppose it's not intended to be there

**Review by ColCh — CHANGES_REQUESTED** [review]
Great! But needs 3 fixes

File: base (hunk 3 above)
Comment (keshan3262) [reply base]: It was intended but I've made sure just now that we can do without this symbolic link.

**Comment by keshan3262** [comment]
Tests fail now because of some strange 'evening disease' of Travis. I'm sure they'll be passed tomorrow.

**Comment by ColCh** [comment]
@keshan3262 great but now remove please `.eslintcache` from git index

See `50-fix-tests-errors`: https://github.com/Lingvokot/lingvokot.github.io/blob/50-fix-tests-errors/.eslintcache . It's up in there, event ignored

Just run `git rm .eslintcache` and commit it

**Review by ColCh — APPROVED** [review]

**Comment by ColCh** [comment]
See, build passed now. Coincidence? Who knows

**Pull request merged**